<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Offlineshop</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/64x64/offlineshop.png</normaloff>:/icons/64x64/offlineshop.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_pricelist">
       <attribute name="title">
        <string>Pricelist</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditFilter"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="tableViewPricelist">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_customer">
       <attribute name="title">
        <string>Customer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditFilterCustomer"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="tableViewCustomer">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCarts">
       <attribute name="title">
        <string>Cart</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Database"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionSave_Database"/>
    <addaction name="actionClose_Database"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFill_with_Example_Data"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_Database"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Entry_Pricelist"/>
   <addaction name="actionDelete_Entry"/>
   <addaction name="actionRevert_Changes"/>
   <addaction name="actionShow_Carts"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Cart"/>
   <addaction name="actionAdd_to_Cart"/>
   <addaction name="actionRemove_From_Cart"/>
   <addaction name="actionExportODF"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/document-new.png</normaloff>:/icons/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/document-open.png</normaloff>:/icons/16x16/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/document-save.png</normaloff>:/icons/16x16/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Database</string>
   </property>
  </action>
  <action name="actionClose_Database">
   <property name="text">
    <string>Close Database</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/process-stop.png</normaloff>:/icons/16x16/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRevert_Changes">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/edit-clear.png</normaloff>:/icons/16x16/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Revert Changes</string>
   </property>
  </action>
  <action name="actionNew_Entry_Pricelist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/list-add.png</normaloff>:/icons/16x16/list-add.png</iconset>
   </property>
   <property name="text">
    <string>New Entry</string>
   </property>
  </action>
  <action name="actionFill_with_Example_Data">
   <property name="text">
    <string>Fill with Example Data</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/list-remove.png</normaloff>:/icons/16x16/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </action>
  <action name="actionExportODF">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/format-indent-more.png</normaloff>:/icons/16x16/format-indent-more.png</iconset>
   </property>
   <property name="text">
    <string>Export Word</string>
   </property>
   <property name="toolTip">
    <string>Export Word</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAdd_to_Cart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/go-next.png</normaloff>:/icons/16x16/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Add to Cart</string>
   </property>
  </action>
  <action name="actionRemove_From_Cart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/go-previous.png</normaloff>:/icons/16x16/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Remove From Cart</string>
   </property>
  </action>
  <action name="actionShow_Carts">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/window-new.png</normaloff>:/icons/16x16/window-new.png</iconset>
   </property>
   <property name="text">
    <string>Show Carts</string>
   </property>
  </action>
  <action name="actionNew_Cart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/offlineshop.png</normaloff>:/icons/16x16/offlineshop.png</iconset>
   </property>
   <property name="text">
    <string>New Cart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
